components/mosart/cime_config/config_compsets.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+
+<compsets>
+
+  <help>
+  Compset format is explained in cime/config/e3sm/allactive/config_compsets.xml
+  or in the CIME documenation
+  </help>
+
+  <!---MOSART RM compsets -->
+
+  <compset>
+    <alias>RM</alias>
```

Comment:
How about we change this alias to something more conventional? For example "RMOSARTNLDAS" - "R" for River case, "MOSART" for the model name, and "NLDAS" for the grid?

Reply:
> I'm fine with the current ones but I will let Zeli decide.

I am fine with the current one too.